src/TypeScript/Analyzer/EslintBridgeAnalyzer.cs

```diff
@@ -70,28 +70,22 @@ internal sealed class EslintBridgeAnalyzer : IEslintBridgeAnalyzer
         }
 
         public async Task<IReadOnlyCollection<IAnalysisIssue>> Analyze(string filePath, string tsConfig, CancellationToken cancellationToken)
-        {
-            try
-            {
-                return await GetFileIssues(filePath, tsConfig, cancellationToken);
-            }
-            catch (EslintBridgeClientNotInitializedException)
-            {
-                RequireLinterUpdate();
-                return await GetFileIssues(filePath, tsConfig, cancellationToken);
-            }
-        }
-
-        private async Task<IReadOnlyCollection<IAnalysisIssue>> GetFileIssues(string filePath, string tsConfig, CancellationToken cancellationToken)
         {
             await EnsureEslintBridgeClientIsInitialized(rulesProvider.GetActiveRulesConfiguration(), cancellationToken);
-
             var analysisResponse = await eslintBridgeClient.Analyze(filePath, tsConfig, cancellationToken);
 
             if (analysisResponse.ParsingError != null)
             {
-                LogParsingError(filePath, analysisResponse.ParsingError);
-                return Array.Empty<IAnalysisIssue>();
+                if (analysisResponse.ParsingError.Message.Contains(Resources.ERR_ParsingError_LinterNotInitialized))
+                {
+                    await eslintBridgeClient.InitLinter(rulesProvider.GetActiveRulesConfiguration(), cancellationToken);
```

Comment:
I thought about using recursion and just re-invoking this method, but I was worried about somehow getting stuck in an infinite loop. So I'm just directly calling  `eslintBridgeClient.InitLinter` and `eslintBridgeClient.Analyze`.

Reply:
The flow isn't quite right - if the second call to `analyze` returns a parsing error then it won't be logged. Logically a parsing failure in the second "analyze" should be handled the same as if it occurred in the first "analyze" i.e.

```
response = await analyze;
if (linter is not initialized)
{
  await init-linter;
  response = await analyze;
}

if (parsing error in response)
{
 ...
}

```

Reply:
Woops, you're right. Nice catch 👍